==29853== Memcheck, a memory error detector
==29853== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==29853== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==29853== Command: /home/wasmer/repositories/wasmer/wasmer/target/debug/examples/instance
==29853== 
==29853== 
==29853== HEAP SUMMARY:
==29853==     in use at exit: 23,088 bytes in 82 blocks
==29853==   total heap usage: 535 allocs, 453 frees, 97,294 bytes allocated
==29853== 
==29853== 8 bytes in 1 blocks are still reachable in loss record 1 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xD5F3CB: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xD601F3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xD5F32C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD30B92: new<alloc::sync::Arc<rayon_core::registry::Registry>> (boxed.rs:177)
==29853==    by 0xD30B92: rayon_core::util::leak (util.rs:5)
==29853==    by 0xD35E8E: rayon_core::registry::set_global_registry::{{closure}}::{{closure}} (registry.rs:196)
==29853==    by 0xD6BEC8: core::result::Result<T,E>::map (result.rs:519)
==29853==    by 0xD35F29: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29853==    by 0xD593E3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853==    by 0x1C16979: std::sync::once::Once::call_inner (once.rs:416)
==29853==    by 0xD59360: std::sync::once::Once::call_once (once.rs:264)
==29853==    by 0xD35E36: rayon_core::registry::set_global_registry (registry.rs:194)
==29853== 
==29853== 32 bytes in 1 blocks are still reachable in loss record 2 of 30
==29853==    at 0x4C33B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x4E3F7F4: _dlerror_run (dlerror.c:140)
==29853==    by 0x4E3F165: dlsym (dlsym.c:70)
==29853==    by 0x1C2B8A3: fetch (weak.rs:61)
==29853==    by 0x1C2B8A3: get<unsafe extern "C" fn(*const libc::unix::linux_like::linux::gnu::b64::x86_64::pthread_attr_t) -> usize> (weak.rs:46)
==29853==    by 0x1C2B8A3: min_stack_size (thread.rs:449)
==29853==    by 0x1C2B8A3: std::sys::unix::thread::Thread::new (thread.rs:49)
==29853==    by 0xD658CB: std::thread::Builder::spawn_unchecked (mod.rs:492)
==29853==    by 0xD66AFB: std::thread::Builder::spawn (mod.rs:386)
==29853==    by 0xD35918: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29853==    by 0xD369FF: rayon_core::registry::Registry::new (registry.rs:260)
==29853==    by 0xD35B8D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29853==    by 0xD35F19: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29853==    by 0xD593E3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853==    by 0x1C16979: std::sync::once::Once::call_inner (once.rs:416)
==29853== 
==29853== 64 bytes in 2 blocks are still reachable in loss record 3 of 30
==29853==    at 0x4C33B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x56C48E1: __cxa_thread_atexit_impl (cxa_thread_atexit_impl.c:106)
==29853==    by 0x1BF7E2E: register_dtor (fast_thread_local.rs:33)
==29853==    by 0x1BF7E2E: try_register_dtor<core::cell::RefCell<core::option::Option<std::sys_common::thread_info::ThreadInfo>>> (local.rs:438)
==29853==    by 0x1BF7E2E: std::thread::local::fast::Key<T>::try_initialize (local.rs:424)
==29853==    by 0x1C1A629: get<core::cell::RefCell<core::option::Option<std::sys_common::thread_info::ThreadInfo>>,fn() -> core::cell::RefCell<core::option::Option<std::sys_common::thread_info::ThreadInfo>>> (local.rs:410)
==29853==    by 0x1C1A629: __getit (local.rs:175)
==29853==    by 0x1C1A629: try_with<core::cell::RefCell<core::option::Option<std::sys_common::thread_info::ThreadInfo>>,closure-0,()> (local.rs:262)
==29853==    by 0x1C1A629: with<core::cell::RefCell<core::option::Option<std::sys_common::thread_info::ThreadInfo>>,closure-0,()> (local.rs:239)
==29853==    by 0x1C1A629: std::sys_common::thread_info::set (thread_info.rs:40)
==29853==    by 0xD65C91: std::thread::Builder::spawn_unchecked::{{closure}} (mod.rs:473)
==29853==    by 0xD5A3AE: core::ops::function::FnOnce::call_once{{vtable-shim}} (function.rs:233)
==29853==    by 0x1C2BC19: call_once<(),FnOnce<()>> (boxed.rs:1081)
==29853==    by 0x1C2BC19: call_once<(),alloc::boxed::Box<FnOnce<()>>> (boxed.rs:1081)
==29853==    by 0x1C2BC19: std::sys::unix::thread::Thread::new::thread_start (thread.rs:87)
==29853==    by 0x52516DA: start_thread (pthread_create.c:463)
==29853==    by 0x57A271E: clone (clone.S:95)
==29853== 
==29853== 64 bytes in 2 blocks are still reachable in loss record 4 of 30
==29853==    at 0x4C33B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x56C48E1: __cxa_thread_atexit_impl (cxa_thread_atexit_impl.c:106)
==29853==    by 0xD9DC01: std::thread::local::fast::Key<T>::try_register_dtor (local.rs:438)
==29853==    by 0xD9DB02: std::thread::local::fast::Key<T>::try_initialize (local.rs:424)
==29853==    by 0xD9DCA6: std::thread::local::fast::Key<T>::get (local.rs:410)
==29853==    by 0xD9D9FC: crossbeam_epoch::default::HANDLE::__getit (local.rs:175)
==29853==    by 0xD476A4: std::thread::local::LocalKey<T>::try_with (local.rs:262)
==29853==    by 0xD4D032: crossbeam_epoch::default::with_handle (default.rs:43)
==29853==    by 0xD4D316: crossbeam_epoch::default::is_pinned (default.rs:30)
==29853==    by 0xD4313C: crossbeam_deque::deque::Stealer<T>::steal (deque.rs:619)
==29853==    by 0xD38CE2: rayon_core::registry::WorkerThread::steal::{{closure}} (registry.rs:779)
==29853==    by 0xD340CC: core::iter::traits::iterator::Iterator::find_map::check::{{closure}} (iterator.rs:2258)
==29853== 
==29853== 64 bytes in 4 blocks are still reachable in loss record 5 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xD5F3CB: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xD601F3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xD5F32C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD44876: new<crossbeam_deque::deque::Buffer<rayon_core::job::JobRef>> (boxed.rs:177)
==29853==    by 0xD44876: <T as crossbeam_epoch::atomic::Pointable>::init (atomic.rs:188)
==29853==    by 0xD61123: crossbeam_epoch::atomic::Owned<T>::init (atomic.rs:861)
==29853==    by 0xD61273: crossbeam_epoch::atomic::Atomic<T>::init (atomic.rs:309)
==29853==    by 0xD61243: crossbeam_epoch::atomic::Atomic<T>::new (atomic.rs:294)
==29853==    by 0xD42F76: crossbeam_deque::deque::Worker<T>::new_lifo (deque.rs:253)
==29853==    by 0xD36E7E: rayon_core::registry::Registry::new::{{closure}} (registry.rs:229)
==29853==    by 0xD6FB8C: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29853==    by 0xD407DD: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29853== 
==29853== 64 bytes in 4 blocks are still reachable in loss record 6 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x1C2B7FC: alloc (alloc.rs:80)
==29853==    by 0x1C2B7FC: alloc (alloc.rs:174)
==29853==    by 0x1C2B7FC: exchange_malloc (alloc.rs:277)
==29853==    by 0x1C2B7FC: std::sys::unix::thread::Thread::new (thread.rs:44)
==29853==    by 0xD658CB: std::thread::Builder::spawn_unchecked (mod.rs:492)
==29853==    by 0xD66AFB: std::thread::Builder::spawn (mod.rs:386)
==29853==    by 0xD35918: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29853==    by 0xD369FF: rayon_core::registry::Registry::new (registry.rs:260)
==29853==    by 0xD35B8D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29853==    by 0xD35F19: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29853==    by 0xD593E3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853==    by 0x1C16979: std::sync::once::Once::call_inner (once.rs:416)
==29853==    by 0xD59360: std::sync::once::Once::call_once (once.rs:264)
==29853==    by 0xD35E36: rayon_core::registry::set_global_registry (registry.rs:194)
==29853== 
==29853== 72 bytes in 1 blocks are still reachable in loss record 7 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x181DF8B: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0x181E0B3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0x181DEEC: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0x1820DA7: std::sync::rwlock::RwLock<T>::new (rwlock.rs:134)
==29853==    by 0x18210DC: <std::sync::rwlock::RwLock<T> as core::default::Default>::default (rwlock.rs:451)
==29853==    by 0x1812A50: __static_ref_initialize (frame_info.rs:28)
==29853==    by 0x1812A50: core::ops::function::FnOnce::call_once (function.rs:233)
==29853==    by 0x184A76B: lazy_static::lazy::Lazy<T>::get::{{closure}} (inline_lazy.rs:31)
==29853==    by 0x18490E4: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853==    by 0x1C16979: std::sync::once::Once::call_inner (once.rs:416)
==29853==    by 0x1849060: std::sync::once::Once::call_once (once.rs:264)
==29853==    by 0x180DE78: get<std::sync::rwlock::RwLock<wasmer_engine::trap::frame_info::GlobalFrameInfo>,fn() -> std::sync::rwlock::RwLock<wasmer_engine::trap::frame_info::GlobalFrameInfo>> (inline_lazy.rs:30)
==29853==    by 0x180DE78: __stability (lib.rs:142)
==29853==    by 0x180DE78: <wasmer_engine::trap::frame_info::FRAME_INFO as core::ops::deref::Deref>::deref (lib.rs:144)
==29853== 
==29853== 160 bytes in 4 blocks are still reachable in loss record 8 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xD5F3CB: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xD601F3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xD5F32C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD63FDF: std::sync::mutex::Mutex<T>::new (mutex.rs:223)
==29853==    by 0xD777F0: rayon_core::latch::LockLatch::new (latch.rs:230)
==29853==    by 0xD37C2C: rayon_core::registry::ThreadInfo::new (registry.rs:583)
==29853==    by 0xD5A2C1: core::ops::function::FnMut::call_mut (function.rs:154)
==29853==    by 0xD6F8A3: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29853==    by 0xD53243: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29853==    by 0xD71158: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29853==    by 0xD6F059: core::iter::traits::iterator::Iterator::for_each (iterator.rs:658)
==29853== 
==29853== 160 bytes in 4 blocks are still reachable in loss record 9 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xD5F3CB: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xD601F3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xD5F32C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD63FDF: std::sync::mutex::Mutex<T>::new (mutex.rs:223)
==29853==    by 0xD777F0: rayon_core::latch::LockLatch::new (latch.rs:230)
==29853==    by 0xD37C44: rayon_core::registry::ThreadInfo::new (registry.rs:584)
==29853==    by 0xD5A2C1: core::ops::function::FnMut::call_mut (function.rs:154)
==29853==    by 0xD6F8A3: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29853==    by 0xD53243: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29853==    by 0xD71158: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29853==    by 0xD6F059: core::iter::traits::iterator::Iterator::for_each (iterator.rs:658)
==29853== 
==29853== 160 bytes in 4 blocks are still reachable in loss record 10 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xD5F3CB: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xD601F3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xD5F32C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD63FDF: std::sync::mutex::Mutex<T>::new (mutex.rs:223)
==29853==    by 0xD67953: <std::sync::mutex::Mutex<T> as core::default::Default>::default (mutex.rs:446)
==29853==    by 0xD5263E: <rayon_core::sleep::WorkerSleepState as core::default::Default>::default (mod.rs:55)
==29853==    by 0xD4FA6D: <crossbeam_utils::cache_padded::CachePadded<T> as core::default::Default>::default (cache_padded.rs:77)
==29853==    by 0xD50EDA: rayon_core::sleep::Sleep::new::{{closure}} (mod.rs:67)
==29853==    by 0xD6FA50: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29853==    by 0xD405E8: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29853==    by 0xD71229: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29853== 
==29853== 160 bytes in 4 blocks are still reachable in loss record 11 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x1BF91CE: alloc (alloc.rs:80)
==29853==    by 0x1BF91CE: alloc (alloc.rs:174)
==29853==    by 0x1BF91CE: exchange_malloc (alloc.rs:277)
==29853==    by 0x1BF91CE: new<()> (mutex.rs:223)
==29853==    by 0x1BF91CE: std::thread::Thread::new (mod.rs:1149)
==29853==    by 0xD6566D: std::thread::Builder::spawn_unchecked (mod.rs:462)
==29853==    by 0xD66AFB: std::thread::Builder::spawn (mod.rs:386)
==29853==    by 0xD35918: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29853==    by 0xD369FF: rayon_core::registry::Registry::new (registry.rs:260)
==29853==    by 0xD35B8D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29853==    by 0xD35F19: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29853==    by 0xD593E3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853==    by 0x1C16979: std::sync::once::Once::call_inner (once.rs:416)
==29853==    by 0xD59360: std::sync::once::Once::call_once (once.rs:264)
==29853==    by 0xD35E36: rayon_core::registry::set_global_registry (registry.rs:194)
==29853== 
==29853== 160 bytes in 4 blocks are still reachable in loss record 12 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xD5F3CB: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xD601F3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xD5F32C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD2F037: alloc::sync::Arc<T>::new (sync.rs:324)
==29853==    by 0xD6570B: std::thread::Builder::spawn_unchecked (mod.rs:465)
==29853==    by 0xD66AFB: std::thread::Builder::spawn (mod.rs:386)
==29853==    by 0xD35918: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29853==    by 0xD369FF: rayon_core::registry::Registry::new (registry.rs:260)
==29853==    by 0xD35B8D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29853==    by 0xD35F19: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29853==    by 0xD593E3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853== 
==29853== 192 bytes in 4 blocks are still reachable in loss record 13 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x1C162C6: alloc (alloc.rs:80)
==29853==    by 0x1C162C6: alloc (alloc.rs:174)
==29853==    by 0x1C162C6: exchange_malloc (alloc.rs:277)
==29853==    by 0x1C162C6: std::sync::condvar::Condvar::new (condvar.rs:130)
==29853==    by 0xD7780C: rayon_core::latch::LockLatch::new (latch.rs:231)
==29853==    by 0xD37C2C: rayon_core::registry::ThreadInfo::new (registry.rs:583)
==29853==    by 0xD5A2C1: core::ops::function::FnMut::call_mut (function.rs:154)
==29853==    by 0xD6F8A3: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29853==    by 0xD53243: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29853==    by 0xD71158: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29853==    by 0xD6F059: core::iter::traits::iterator::Iterator::for_each (iterator.rs:658)
==29853==    by 0xD56C31: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::spec_extend (vec.rs:2140)
==29853==    by 0xD578B4: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter (vec.rs:2120)
==29853==    by 0xD58856: <alloc::vec::Vec<T> as core::iter::traits::collect::FromIterator<T>>::from_iter (vec.rs:1995)
==29853== 
==29853== 192 bytes in 4 blocks are still reachable in loss record 14 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x1C162C6: alloc (alloc.rs:80)
==29853==    by 0x1C162C6: alloc (alloc.rs:174)
==29853==    by 0x1C162C6: exchange_malloc (alloc.rs:277)
==29853==    by 0x1C162C6: std::sync::condvar::Condvar::new (condvar.rs:130)
==29853==    by 0xD7780C: rayon_core::latch::LockLatch::new (latch.rs:231)
==29853==    by 0xD37C44: rayon_core::registry::ThreadInfo::new (registry.rs:584)
==29853==    by 0xD5A2C1: core::ops::function::FnMut::call_mut (function.rs:154)
==29853==    by 0xD6F8A3: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29853==    by 0xD53243: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29853==    by 0xD71158: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29853==    by 0xD6F059: core::iter::traits::iterator::Iterator::for_each (iterator.rs:658)
==29853==    by 0xD56C31: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::spec_extend (vec.rs:2140)
==29853==    by 0xD578B4: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter (vec.rs:2120)
==29853==    by 0xD58856: <alloc::vec::Vec<T> as core::iter::traits::collect::FromIterator<T>>::from_iter (vec.rs:1995)
==29853== 
==29853== 192 bytes in 4 blocks are still reachable in loss record 15 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x1C162C6: alloc (alloc.rs:80)
==29853==    by 0x1C162C6: alloc (alloc.rs:174)
==29853==    by 0x1C162C6: exchange_malloc (alloc.rs:277)
==29853==    by 0x1C162C6: std::sync::condvar::Condvar::new (condvar.rs:130)
==29853==    by 0xD5265A: <rayon_core::sleep::WorkerSleepState as core::default::Default>::default (mod.rs:57)
==29853==    by 0xD4FA6D: <crossbeam_utils::cache_padded::CachePadded<T> as core::default::Default>::default (cache_padded.rs:77)
==29853==    by 0xD50EDA: rayon_core::sleep::Sleep::new::{{closure}} (mod.rs:67)
==29853==    by 0xD6FA50: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29853==    by 0xD405E8: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29853==    by 0xD71229: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29853==    by 0xD6EE8A: core::iter::traits::iterator::Iterator::for_each (iterator.rs:658)
==29853==    by 0xD56FF7: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::spec_extend (vec.rs:2140)
==29853==    by 0xD57811: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter (vec.rs:2120)
==29853==    by 0xD587F5: <alloc::vec::Vec<T> as core::iter::traits::collect::FromIterator<T>>::from_iter (vec.rs:1995)
==29853== 
==29853== 192 bytes in 4 blocks are still reachable in loss record 16 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x1BF9232: alloc (alloc.rs:80)
==29853==    by 0x1BF9232: alloc (alloc.rs:174)
==29853==    by 0x1BF9232: exchange_malloc (alloc.rs:277)
==29853==    by 0x1BF9232: new (condvar.rs:130)
==29853==    by 0x1BF9232: std::thread::Thread::new (mod.rs:1150)
==29853==    by 0xD6566D: std::thread::Builder::spawn_unchecked (mod.rs:462)
==29853==    by 0xD66AFB: std::thread::Builder::spawn (mod.rs:386)
==29853==    by 0xD35918: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29853==    by 0xD369FF: rayon_core::registry::Registry::new (registry.rs:260)
==29853==    by 0xD35B8D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29853==    by 0xD35F19: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29853==    by 0xD593E3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853==    by 0x1C16979: std::sync::once::Once::call_inner (once.rs:416)
==29853==    by 0xD59360: std::sync::once::Once::call_once (once.rs:264)
==29853==    by 0xD35E36: rayon_core::registry::set_global_registry (registry.rs:194)
==29853== 
==29853== 320 bytes in 4 blocks are still reachable in loss record 17 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x1BF9281: alloc (alloc.rs:80)
==29853==    by 0x1BF9281: alloc (alloc.rs:174)
==29853==    by 0x1BF9281: exchange_malloc (alloc.rs:277)
==29853==    by 0x1BF9281: new<std::thread::Inner> (sync.rs:324)
==29853==    by 0x1BF9281: std::thread::Thread::new (mod.rs:1145)
==29853==    by 0xD6566D: std::thread::Builder::spawn_unchecked (mod.rs:462)
==29853==    by 0xD66AFB: std::thread::Builder::spawn (mod.rs:386)
==29853==    by 0xD35918: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29853==    by 0xD369FF: rayon_core::registry::Registry::new (registry.rs:260)
==29853==    by 0xD35B8D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29853==    by 0xD35F19: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29853==    by 0xD593E3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853==    by 0x1C16979: std::sync::once::Once::call_inner (once.rs:416)
==29853==    by 0xD59360: std::sync::once::Once::call_once (once.rs:264)
==29853==    by 0xD35E36: rayon_core::registry::set_global_registry (registry.rs:194)
==29853== 
==29853== 384 bytes in 1 blocks are still reachable in loss record 18 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xDC656B: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xDC6E13: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xDBD92A: alloc::raw_vec::finish_grow (raw_vec.rs:497)
==29853==    by 0xD7A5B0: alloc::raw_vec::RawVec<T,A>::grow_amortized (raw_vec.rs:429)
==29853==    by 0xD78F03: alloc::raw_vec::RawVec<T,A>::try_reserve (raw_vec.rs:310)
==29853==    by 0xD7CB02: alloc::raw_vec::RawVec<T,A>::reserve (raw_vec.rs:300)
==29853==    by 0xD559E8: alloc::vec::Vec<T>::reserve (vec.rs:505)
==29853==    by 0xD56B47: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::spec_extend (vec.rs:2136)
==29853==    by 0xD578B4: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter (vec.rs:2120)
==29853==    by 0xD58856: <alloc::vec::Vec<T> as core::iter::traits::collect::FromIterator<T>>::from_iter (vec.rs:1995)
==29853==    by 0xD6ECB8: core::iter::traits::iterator::Iterator::collect (iterator.rs:1671)
==29853== 
==29853== 416 bytes in 4 blocks are still reachable in loss record 19 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xD5F3CB: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xD601F3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xD5F32C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD65837: new<closure-0> (boxed.rs:177)
==29853==    by 0xD65837: std::thread::Builder::spawn_unchecked (mod.rs:494)
==29853==    by 0xD66AFB: std::thread::Builder::spawn (mod.rs:386)
==29853==    by 0xD35918: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29853==    by 0xD369FF: rayon_core::registry::Registry::new (registry.rs:260)
==29853==    by 0xD35B8D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29853==    by 0xD35F19: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29853==    by 0xD593E3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853==    by 0x1C16979: std::sync::once::Once::call_inner (once.rs:416)
==29853== 
==29853== 512 bytes in 1 blocks are still reachable in loss record 20 of 30
==29853==    at 0x4C33E76: memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x4C33F91: posix_memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x1C1BD39: aligned_malloc (alloc.rs:95)
==29853==    by 0x1C1BD39: alloc (alloc.rs:22)
==29853==    by 0x1C1BD39: __rdl_alloc (alloc.rs:304)
==29853==    by 0xDC656B: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xDC6E13: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xDBD92A: alloc::raw_vec::finish_grow (raw_vec.rs:497)
==29853==    by 0xD7A910: alloc::raw_vec::RawVec<T,A>::grow_amortized (raw_vec.rs:429)
==29853==    by 0xD78E03: alloc::raw_vec::RawVec<T,A>::try_reserve (raw_vec.rs:310)
==29853==    by 0xD7CA62: alloc::raw_vec::RawVec<T,A>::reserve (raw_vec.rs:300)
==29853==    by 0xD55988: alloc::vec::Vec<T>::reserve (vec.rs:505)
==29853==    by 0xD56F37: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::spec_extend (vec.rs:2136)
==29853==    by 0xD57811: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter (vec.rs:2120)
==29853== 
==29853== 640 bytes in 1 blocks are still reachable in loss record 21 of 30
==29853==    at 0x4C33E76: memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x4C33F91: posix_memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x1C1BD39: aligned_malloc (alloc.rs:95)
==29853==    by 0x1C1BD39: alloc (alloc.rs:22)
==29853==    by 0x1C1BD39: __rdl_alloc (alloc.rs:304)
==29853==    by 0xD5F3CB: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xD601F3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xD5F32C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD2EE65: alloc::sync::Arc<T>::new (sync.rs:324)
==29853==    by 0xD365F7: rayon_core::registry::Registry::new (registry.rs:238)
==29853==    by 0xD35B8D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29853==    by 0xD35F19: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29853==    by 0xD593E3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853==    by 0x1C16979: std::sync::once::Once::call_inner (once.rs:416)
==29853== 
==29853== 640 bytes in 1 blocks are still reachable in loss record 22 of 30
==29853==    at 0x4C33E76: memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x4C33F91: posix_memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x1C1BD39: aligned_malloc (alloc.rs:95)
==29853==    by 0x1C1BD39: alloc (alloc.rs:22)
==29853==    by 0x1C1BD39: __rdl_alloc (alloc.rs:304)
==29853==    by 0xDA246B: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xDA29B3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xDA23CC: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD9E645: alloc::sync::Arc<T>::new (sync.rs:324)
==29853==    by 0xD9EB89: <crossbeam_epoch::collector::Collector as core::default::Default>::default (collector.rs:32)
==29853==    by 0xD9EBB6: crossbeam_epoch::collector::Collector::new (collector.rs:40)
==29853==    by 0xD9D0A9: __static_ref_initialize (default.rs:13)
==29853==    by 0xD9D0A9: core::ops::function::FnOnce::call_once (function.rs:233)
==29853==    by 0xD9CF15: lazy_static::lazy::Lazy<T>::get::{{closure}} (inline_lazy.rs:31)
==29853==    by 0xD9E1D4: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853== 
==29853== 808 bytes in 1 blocks are still reachable in loss record 23 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x181DF8B: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0x181E0B3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0x181DEEC: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0x182C160: new<alloc::collections::btree::node::LeafNode<usize, wasmer_engine::trap::frame_info::ModuleInfoFrameInfo>> (boxed.rs:177)
==29853==    by 0x182C160: alloc::collections::btree::node::Root<K,V>::new_leaf (node.rs:157)
==29853==    by 0x1812A69: core::ops::function::FnOnce::call_once (function.rs:233)
==29853==    by 0x181F1AC: core::option::Option<T>::get_or_insert_with (option.rs:853)
==29853==    by 0x1844A6E: alloc::collections::btree::map::BTreeMap<K,V>::ensure_root_is_owned (map.rs:2324)
==29853==    by 0x1844E41: alloc::collections::btree::map::BTreeMap<K,V>::entry (map.rs:1061)
==29853==    by 0x1845CB9: alloc::collections::btree::map::BTreeMap<K,V>::insert (map.rs:829)
==29853==    by 0x180DB8F: wasmer_engine::trap::frame_info::register (frame_info.rs:281)
==29853==    by 0xBD6F64: <wasmer_engine_jit::artifact::JITArtifact as wasmer_engine::artifact::Artifact>::register_frame_info (artifact.rs:275)
==29853== 
==29853== 1,152 bytes in 4 blocks are possibly lost in loss record 24 of 30
==29853==    at 0x4C33B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x4013666: allocate_dtv (dl-tls.c:286)
==29853==    by 0x4013666: _dl_allocate_tls (dl-tls.c:530)
==29853==    by 0x5252227: allocate_stack (allocatestack.c:627)
==29853==    by 0x5252227: pthread_create@@GLIBC_2.2.5 (pthread_create.c:644)
==29853==    by 0x1C2B94C: std::sys::unix::thread::Thread::new (thread.rs:66)
==29853==    by 0xD658CB: std::thread::Builder::spawn_unchecked (mod.rs:492)
==29853==    by 0xD66AFB: std::thread::Builder::spawn (mod.rs:386)
==29853==    by 0xD35918: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29853==    by 0xD369FF: rayon_core::registry::Registry::new (registry.rs:260)
==29853==    by 0xD35B8D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29853==    by 0xD35F19: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29853==    by 0xD593E3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853==    by 0x1C16979: std::sync::once::Once::call_inner (once.rs:416)
==29853== 
==29853== 1,520 bytes in 1 blocks are still reachable in loss record 25 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xD5F3CB: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xD601F3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xD5F32C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD44C51: new<crossbeam_deque::deque::Block<rayon_core::job::JobRef>> (boxed.rs:177)
==29853==    by 0xD44C51: <crossbeam_deque::deque::Injector<T> as core::default::Default>::default (deque.rs:1187)
==29853==    by 0xD434DD: crossbeam_deque::deque::Injector<T>::new (deque.rs:1213)
==29853==    by 0xD363C3: rayon_core::registry::Registry::new (registry.rs:242)
==29853==    by 0xD35B8D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29853==    by 0xD35F19: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29853==    by 0xD593E3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29853==    by 0x1C16979: std::sync::once::Once::call_inner (once.rs:416)
==29853==    by 0xD59360: std::sync::once::Once::call_once (once.rs:264)
==29853== 
==29853== 1,536 bytes in 4 blocks are still reachable in loss record 26 of 30
==29853==    at 0x4C33E76: memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x4C33F91: posix_memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0x1C1BD39: aligned_malloc (alloc.rs:95)
==29853==    by 0x1C1BD39: alloc (alloc.rs:22)
==29853==    by 0x1C1BD39: __rdl_alloc (alloc.rs:304)
==29853==    by 0xD5F3CB: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xD601F3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xD5F32C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD2EC85: alloc::sync::Arc<T>::new (sync.rs:324)
==29853==    by 0xD4300D: crossbeam_deque::deque::Worker<T>::new_lifo (deque.rs:250)
==29853==    by 0xD36E7E: rayon_core::registry::Registry::new::{{closure}} (registry.rs:229)
==29853==    by 0xD6FB8C: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29853==    by 0xD407DD: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29853==    by 0xD713E3: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29853== 
==29853== 2,008 bytes in 1 blocks are still reachable in loss record 27 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xDA246B: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xDA29B3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xDA23CC: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD9AD89: new<crossbeam_epoch::sync::queue::Node<crossbeam_epoch::internal::SealedBag>> (boxed.rs:177)
==29853==    by 0xD9AD89: <T as crossbeam_epoch::atomic::Pointable>::init (atomic.rs:188)
==29853==    by 0xD9A735: crossbeam_epoch::atomic::Owned<T>::init (atomic.rs:861)
==29853==    by 0xD9A685: crossbeam_epoch::atomic::Owned<T>::new (atomic.rs:846)
==29853==    by 0xD9AF8B: crossbeam_epoch::sync::queue::Queue<T>::new (queue.rs:50)
==29853==    by 0xD96383: crossbeam_epoch::internal::Global::new (internal.rs:254)
==29853==    by 0xD9EB7B: <crossbeam_epoch::collector::Collector as core::default::Default>::default (collector.rs:32)
==29853==    by 0xD9EBB6: crossbeam_epoch::collector::Collector::new (collector.rs:40)
==29853==    by 0xD9D0A9: __static_ref_initialize (default.rs:13)
==29853==    by 0xD9D0A9: core::ops::function::FnOnce::call_once (function.rs:233)
==29853== 
==29853== 3,040 bytes in 2 blocks are still reachable in loss record 28 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xD5F3CB: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xD601F3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xD5F32C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD44C51: new<crossbeam_deque::deque::Block<rayon_core::job::JobRef>> (boxed.rs:177)
==29853==    by 0xD44C51: <crossbeam_deque::deque::Injector<T> as core::default::Default>::default (deque.rs:1187)
==29853==    by 0xD434DD: crossbeam_deque::deque::Injector<T>::new (deque.rs:1213)
==29853==    by 0xD30A5C: rayon_core::job::JobFifo::new (job.rs:193)
==29853==    by 0xD38E1A: rayon_core::registry::main_loop (registry.rs:806)
==29853==    by 0xD3544F: rayon_core::registry::ThreadBuilder::run (registry.rs:56)
==29853==    by 0xD35B00: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn::{{closure}} (registry.rs:101)
==29853==    by 0xD6E3AF: std::sys_common::backtrace::__rust_begin_short_backtrace (backtrace.rs:130)
==29853==    by 0xD66980: std::thread::Builder::spawn_unchecked::{{closure}}::{{closure}} (mod.rs:475)
==29853== 
==29853== 4,080 bytes in 2 blocks are still reachable in loss record 29 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xDA246B: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xDA29B3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xDA23CC: alloc::alloc::exchange_malloc (alloc.rs:277)
==29853==    by 0xD95DB9: new<crossbeam_epoch::internal::Local> (boxed.rs:177)
==29853==    by 0xD95DB9: <T as crossbeam_epoch::atomic::Pointable>::init (atomic.rs:188)
==29853==    by 0xD9A6D5: crossbeam_epoch::atomic::Owned<T>::init (atomic.rs:861)
==29853==    by 0xD9A635: crossbeam_epoch::atomic::Owned<T>::new (atomic.rs:846)
==29853==    by 0xD96C22: crossbeam_epoch::internal::Local::register (internal.rs:392)
==29853==    by 0xD9EBDD: crossbeam_epoch::collector::Collector::register (collector.rs:45)
==29853==    by 0xD9D9C3: crossbeam_epoch::default::HANDLE::__init (default.rs:18)
==29853==    by 0xD9D088: core::ops::function::FnOnce::call_once (function.rs:233)
==29853==    by 0xDA2B5B: std::thread::local::lazy::LazyKeyInner<T>::initialize (local.rs:289)
==29853== 
==29853== 4,096 bytes in 4 blocks are still reachable in loss record 30 of 30
==29853==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29853==    by 0xD5F3CB: alloc::alloc::alloc (alloc.rs:80)
==29853==    by 0xD601F3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29853==    by 0xD78BBD: alloc::raw_vec::RawVec<T,A>::allocate_in (raw_vec.rs:183)
==29853==    by 0xD7BE30: alloc::raw_vec::RawVec<T,A>::with_capacity_in (raw_vec.rs:159)
==29853==    by 0xD77FAE: alloc::raw_vec::RawVec<T>::with_capacity (raw_vec.rs:90)
==29853==    by 0xD537BE: alloc::vec::Vec<T>::with_capacity (vec.rs:363)
==29853==    by 0xD41A20: crossbeam_deque::deque::Buffer<T>::alloc (deque.rs:45)
==29853==    by 0xD42EF9: crossbeam_deque::deque::Worker<T>::new_lifo (deque.rs:248)
==29853==    by 0xD36E7E: rayon_core::registry::Registry::new::{{closure}} (registry.rs:229)
==29853==    by 0xD6FB8C: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29853==    by 0xD407DD: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29853== 
==29853== LEAK SUMMARY:
==29853==    definitely lost: 0 bytes in 0 blocks
==29853==    indirectly lost: 0 bytes in 0 blocks
==29853==      possibly lost: 1,152 bytes in 4 blocks
==29853==    still reachable: 21,936 bytes in 78 blocks
==29853==         suppressed: 0 bytes in 0 blocks
==29853== 
==29853== For counts of detected and suppressed errors, rerun with: -v
==29853== ERROR SUMMARY: 1 errors from 1 contexts (suppressed: 0 from 0)
